scripts/build_kernel.sh

```diff
@@ -1,35 +1,47 @@
 #!/usr/bin/env bash
 
+SUPPORTED_FLAVOURS=( v8 2712 )
+FLAVOUR=$1
+
+case " ${SUPPORTED_FLAVOURS[@]} " in
+    (*" $FLAVOUR "*) echo "Building for kernel version: $FLAVOUR" ;;
+    (*)
+        echo "Kernel version not supported: $FLAVOUR"
+        echo "Supported versions: ${SUPPORTED_FLAVOURS[@]}"
+        exit 1
+        ;;
+
+esac
```

Comment:
Instead of forcing a parameter for a single flavour, can you default to all flavours being built? We can keep the parameter as a filter (eventually error out if an invalid flavour is submitted)

Something along the lines of:

```bash
SUPPORTED_FLAVOURS=( v8 2712 )
FLAVOUR=$1

# validate flavour / default to SUPPORTED_FLAVOURS if not submitted
REQUESTED_FLAVOURS=( $FLAVOUR )

Then cycle the build procedure for the requested flavours
```

Reply:
Agreed I like this approach more, has been addressed ✅